app/coffeescripts/views/assignments/VeriCiteSettingsDialog.coffee

```diff
@@ -0,0 +1,68 @@
+define [
```

Comment:
I agree... this was too aggressively copied. I'll try to combine the two and at a minimum remove the unused TII code